src/Compilers/Core/MSBuildTask/Microsoft.Build.Tasks.CodeAnalysis.csproj

```diff
@@ -36,6 +36,10 @@
     <Compile Include="..\..\Shared\NamedPipeUtil.cs" />
     <Compile Include="..\..\Shared\BuildServerConnection.cs" />
     <Compile Include="..\..\Shared\RuntimeHostInfo.cs" />
+    <Compile Include="..\CommandLine\BuildProtocol.cs" />
+    <Compile Include="..\CommandLine\ConsoleUtil.cs" />
+    <Compile Include="..\CommandLine\NativeMethods.cs" />
+    <Compile Include="..\CommandLine\CompilerServerLogger.cs" />
```

Comment:
I see that those four `Compile` items are the only things you transferred over from the now removed `CommandLine.projitems` import. 
Did other properties (like `Import_RootNamespace`) not need to be copied here as well?